API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
```

Comment:
Can we do something to allow control over the order an Activity appears in the Lesson Plan? e.g. LessonPlanIndex, so that I can, say, put my six activities in a particular order within the lesson plan? 

If not, what order will they appear in? Chron by creation date?